ParcialProgramacion4/src/Empleado.java

```diff
@@ -29,5 +29,12 @@ public Empleado(
         this.fechaDeIngreso = fechaDeIngreso;
     }
 
-    
+    public boolean hasEmpleado(int legajo) {
```

Comment:
Es el nombre correcto
```suggestion
    public boolean hasLegajo(int legajo) {
```